products/docker-engine.md

```diff
@@ -51,7 +51,7 @@ releases:
 
 -   releaseCycle: "25.0"
     releaseDate: 2024-01-19
-    eol: false
+    eol: 2024-07-25
```

Comment:
The 25.0 branch is not going to be EOL for a couple of years, with Mirantis and Amazon employees contributing backports and CVE fixes

Reply:
Could you update https://github.com/moby/moby/blob/master/project/BRANCHES-AND-TAGS.md

Reply:
https://github.com/moby/moby/pull/48474